### PR TITLE
Overall improvements

### PR DESCRIPTION
**Pull request by filfreire**
## What was done
- Improve and update homepage;
- Fix past events page and create a proper list with past events;
- Improve readme;
- Move @filfreire to hof page;
- Remove gemfile.lock - can cause local conflicts for contributors, we'll need to follow-up this later;

/cc @FabioCB @filipemcarvalho @nmatos @tjbfernandes 

## Screenshots

screen shot 2018-07-26 at 18 27 42
---
screen shot 2018-07-26 at 18 27 52
---
screen shot 2018-07-26 at 18 27 59
---
screen shot 2018-07-26 at 18 28 12
---
screen shot 2018-07-26 at 18 28 18
---
screen shot 2018-07-26 at 18 28 26

### CODE REVIEW

**Comment by filfreire** [comment]
@FabioCB @filipemcarvalho kindly CR 👍

File: pages/hof.md [hunk 1]
```diff
@@ -1,21 +1,34 @@
...
+      <div class="speakermiddle">
+        <div class="speakertext" onclick="on('dj')">Diogo Júnior</div>
+      </div>
+    </div>
+  </div>
+  <div class="column" style="padding: 0.5rem 1rem; border: 1px solid #e9ebec; width: 33%;">
+    <div class="speakercontainer">
+      <img class="speakerimage" src="https://avatars1.githubusercontent.com/u/11976836?s=460&v=4" style="min-width:40px;"/>
+      <div class="speakermiddle">
+         <div class="speakertext" onclick="on('filfreire')">Filipe Freire</div>
```
Comment (filipemcarvalho) [inline pages/hof.md:18]: 👏

File: pages/past_events.md [hunk 2]
```diff
@@ -1,13 +1,82 @@
-<h1> Past Events </h1>
+# Past Events
```
Comment (filipemcarvalho) [inline pages/past_events.md:1]: Great work here 👍

File: Gemfile.lock [hunk 3]
```diff
@@ -1,238 +0,0 @@
-GEM
```
Comment (filipemcarvalho) [inline Gemfile.lock]: Can you give me more details on the problems arised with Gemfile.lock? It is important to have it on version control to make sure when we install the dependencies to run locally we are all on the same page. A reference on this: https://bundler.io/v1.12/bundler_workflow.html

**Review by filipemcarvalho — CHANGES_REQUESTED** [review]
Thanks for these great changes! 
Please check my comment on the Gemfile.lock removal 👍

File: pages/hof.md (hunk 1 above)
Comment (filfreire) [reply pages/hof.md:18]: 👍

File: pages/past_events.md (hunk 2 above)
Comment (filfreire) [reply pages/past_events.md:1]: 👍

File: Gemfile.lock (hunk 3 above)
Comment (filfreire) [reply Gemfile.lock]: @filipemcarvalho `nokogiri` dependency installation froze everything I ran `bundle install`. I searched about it but only found unrelated similar problems dating back to 2014-2015.
Maybe you can merge this PR, and then add the `Gemfile.lock` yourself afterwards?

**Comment by filfreire** [comment]
@filipemcarvalho I reverted the delete of the `Gemfile.lock` file.

Also, just so you have an example of the issue, after I run `gem install github-pages`

```
/Library/Ruby/Gems/2.3.0/gems/bundler-1.16.2/lib/bundler/spec_set.rb:91:in `block in materialize': Could not find commonmarker-0.17.9 in any of the sources (Bundler::GemNotFound)
	from /Library/Ruby/Gems/2.3.0/gems/bundler-1.16.2/lib/bundler/spec_set.rb:85:in `map!'
	from /Library/Ruby/Gems/2.3.0/gems/bundler-1.16.2/lib/bundler/spec_set.rb:85:in `materialize'
	from /Library/Ruby/Gems/2.3.0/gems/bundler-1.16.2/lib/bundler/definition.rb:171:in `specs'
	from /Library/Ruby/Gems/2.3.0/gems/bundler-1.16.2/lib/bundler/definition.rb:238:in `specs_for'
	from /Library/Ruby/Gems/2.3.0/gems/bundler-1.16.2/lib/bundler/definition.rb:227:in `requested_specs'
	from /Library/Ruby/Gems/2.3.0/gems/bundler-1.16.2/lib/bundler/runtime.rb:108:in `block in definition_method'
	from /Library/Ruby/Gems/2.3.0/gems/bundler-1.16.2/lib/bundler/runtime.rb:20:in `setup'
	from /Library/Ruby/Gems/2.3.0/gems/bundler-1.16.2/lib/bundler.rb:107:in `setup'
	from /Library/Ruby/Gems/2.3.0/gems/jekyll-3.7.3/lib/jekyll/plugin_manager.rb:50:in `require_from_bundler'
	from /Library/Ruby/Gems/2.3.0/gems/jekyll-3.7.3/exe/jekyll:11:in `<top (required)>'
	from /usr/local/bin/jekyll:22:in `load'
	from /usr/local/bin/jekyll:22:in `<main>'
```

If I delete the `Gemfile.lock` right after, I'm able to raise the local instance of the site in order to test it.

**Comment by filfreire** [comment]
@filipemcarvalho kindly follow up 😄

**Review by filipemcarvalho — APPROVED** [review]

**Pull request merged**